grails-app/controllers/au/org/ala/profile/api/ApiController.groovy

```diff
@@ -34,6 +34,7 @@ class ApiController extends BaseController {
     MapService mapService
     ApiService apiService
 
+    @RequireApiKey()
```

Comment:
why is requireapikey required? Similarly other below actions.

Reply:
previous ApiController class had @RequireApiKey. so I put it into all method except api/opus/list

Reply:
Keep security standard for all actions.